ui/src/router/components/form/components/experiment_config/EngineTypePanel.js

```diff
@@ -4,8 +4,10 @@ import { EuiForm, EuiFormRow, EuiSuperSelect } from "@elastic/eui";
 import { DescribedFormGroup } from "../../../../../components/form/described_form_group/DescribedFormGroup";
 import { FormLabelWithToolTip } from "../../../../../components/form/label_with_tooltip/FormLabelWithToolTip";
 
-export const EngineTypePanel = ({ type, options, onChange, errors }) => {
+export const EngineTypePanel = ({ type, options, protocol, onChange, errors }) => {
   const selectedOption = options.find((option) => option.value === type);
+  const helpText = protocol === "UPI_V1" ? 
+    "For UPI Router, payload variables should be in Prediction Context" : ""
```

Comment:
"For UPI Router, payload variables should be stored as variable in Prediction Context field of the request payload."